UPGRADING.md

```diff
@@ -1,6 +1,21 @@
 Upgrading Hashie
 ================
 
+### Upgrading to HEAD
+
+One of the features of Hashie::Mash was broken in MRI Ruby versions 2.2.0, 2.2.1, and 2.2.2.  @michaelherold filed the [bug with ruby-core](https://bugs.ruby-lang.org/issues/10985) and it was fixed in the Ruby 2.2.3 release.  The travis build matrix has been updated to 2.2.3 now that it has been released.
+
+The broken behavior was (in `spec/hashie/mash_spec.rb`):
+
+```ruby
+it 'is able to access an unknown suffixed key as a method' do
+  # See https://github.com/intridea/hashie/pull/285 for more information
+  %w(= ? ! _).each do |suffix|
+    expect(subject.method(:"xyz#{suffix}")).to_not be_nil
+  end
+end
+```
```

Comment:
@dblock For clarity:
The pended one reproduces in the spec every time (which is why the pending works - it runs and fails).  There was a second spec that doesn't reproduce every time, only when running the single file, but it isn't pended because it passes when running the whole suite.

I just can't get the currently pended behavior to fail in `irb` at all when running Ruby 2.2.0, which is one of the pended Rubies that it does fail with in the spec.